trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0, 10] if @nameable.correct_name.length > 10
+  end
+end
```

Comment:
- [ ] Please know that the function `correct_name` only returns the name when it's greater than 10, you might want to do an `else` here that covers the case when the `name`'s length is 10 or less, if you test it with a name "John Doe" for example it won't return anything as it is less than 10, kindly fix this.